airbyte-integrations/connectors/source-salesforce/source_salesforce/streams.py

```diff
@@ -352,3 +386,26 @@ class BulkIncrementalSalesforceStream(BulkSalesforceStream, IncrementalSalesforc
     def next_page_token(self, last_record: dict) -> str:
         if self.name not in UNSUPPORTED_FILTERING_STREAMS:
             return last_record[self.cursor_field]
+
+    def request_params(
```

Comment:
Before the change, this method was inherited from `IncrementalSalesforceStream`. In this PR request_params is overriden not to break existing functionality, because parents method was changed.